yaksh/views.py

```diff
@@ -324,6 +324,7 @@ def add_quiz(request, quiz_id=None, course_id=None):
         form = QuizForm(instance=quiz)
         context["quiz_id"] = quiz_id
         context["course_id"] = course_id
+        context["quiz"] = quiz
```

Comment:
A suggestion:
passing quiz and quiz_id to the context is slightly redundant.
You are passing quiz to get question paper in the template, better would be pass question paper id.
This will also help to check whether question paper exists or not, since currently in template if question paper is not found, will get 404 error.